.semaphore/end-to-end/scripts/operator_migrate.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
Code structure wise, I think it would be nice to separate "scripts that are part of generic test running" and "scripts that test functionality" 

This one feels like the script is the test, whereas `global_prologue.sh` is just test setup.

Reply:
Yeah that seems reasonable to me. `scripts/` and `scripts/tests`

Reply:
Added `test_scripts` directory under `scripts` and moved the operator migration script to it.